draft-ietf-rats-corim.md

```diff
@@ -790,25 +791,29 @@ The supply chain entity that is responsible for providing the the measurements (
 is by default the CoRIM signer. If a different entity is authorized to provide measurement values,
 the `authorized-by` statement can be supplied in the `measurement-map`.
 
+An environment may have multiple measurements of the same type.
+Measurement keys may be used to disambiguate these measurements.
```

Comment:
WFM, but another way to put this (which I find more intuitive) is to say that an environment can be composed of multiple individually measured "elements."

I think "element" is the piece of vocabulary we were missing before Dionna proposed it in #257, allowing speaking about multiplicity more naturally.

Quoting from #257:
```
An environment has one or more measured elements.
Measured elements are distinguished from each other by the `measurement-map` key.
A measured element can have a dedicated measurement, such as a security version number.
Multiple elements could be combined into a single measurement, such as a running hash of multiple files.
A measured element can also have several measurement values, such as a name, version, and digest.
Measurements can have class, instance or group scope.
A `measurement-map`'s scope is determined by the `environment-map` it is paired with.
```